src/field/broken-project/index.md

```diff
@@ -1,7 +1,40 @@
 # How to Fix a Broken Project
 [[toc]]
 
-Do you get an error message when trying to open a project in <MobileAppName />? There can be multiple reasons for what went wrong. Here we will try to guide you through some basic steps that can help you.
+Do you get an error message when trying to open a project or form in <MobileAppName />? There can be multiple reasons for what went wrong. Here we will try to guide you through some basic steps that can help you.
+
+:::tip
+Need more help with your issue? <CommunityJoin />
+:::
+
+## Saving of feature failed (vector layer)
+
+<SinceBadge type="App" version="2.5.0" />
+
+The error is caused by a change of behaviour in QGIS versions 3.34.0 and higher as well as <MobileAppName /> 2.5.0 and higher. 
+
+:::details
+You can see the detailed upstream issue report <GitHubRepo id="qgis/QGIS/issues/55517" desc="on this link" />. 
+::: 
+
+The error can be identified by notification *Failed to save changes* in the <MobileAppName /> when a feature is added or modified. 
+![Failed to save changes error](./saving_failed_bug.png "Failed to save changes error")
+connected with the diagnostics log error *Failed to commit changes ... wrong data type for attribute ...*, e.g.
```

Comment:
this  seems like unfinished sentence, maybe:
In the diagnostic log, you may find errors such as Failed to commit changes, or  ... wrong data type for attribute ....